### PR TITLE
README: improvements to Zoom meeting section

### PR DESCRIPTION
**Pull request by axelsimon**
An attempt to make the phone numbers easier to parse.

+ fixes the zoom meeting link formatting.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,26 +86,30 @@ See the  [CNCF Calendar](https://www.cncf.io/calendar/) for calendar invites.
 [Meeting minutes and
 agenda](https://docs.google.com/document/d/170y5biX9k95hYRwprITprG6Mc9xD5glVn-4mB2Jmi2g/)
 
-### Zoom Details
+### Zoom Meeting Details
 
 <!-- cSpell:ignore cncftagsecurity -->
 Meeting Link:
```
Comment (TheFoxAtWork) [inline README.md:92]:
```suggestion
Meeting Link:
```

File: README.md [hunk 2]
```diff
@@ -86,26 +86,30 @@ See the  [CNCF Calendar](https://www.cncf.io/calendar/) for calendar invites.
...
 agenda](https://docs.google.com/document/d/170y5biX9k95hYRwprITprG6Mc9xD5glVn-4mB2Jmi2g/)
 
-### Zoom Details
+### Zoom Meeting Details
 
 <!-- cSpell:ignore cncftagsecurity -->
 Meeting Link: 
-[zoom.us/my/cncftagsecurity (Password: 77777)]
-(https://zoom.us/j/7375677271?pwd=VkxmTjJ6TDVHK29Qb2tQakE4SitWZz09)
+[zoom.us/my/cncftagsecurity](https://zoom.us/j/7375677271?pwd=VkxmTjJ6TDVHK29Qb2tQakE4SitWZz09) (Password: 77777)
```
Comment (TheFoxAtWork) [inline README.md:93]:
```suggestion
[zoom.us/my/cncftagsecurity](https://zoom.us/j/7375677271?pwd=VkxmTjJ6TDVHK29Qb2tQakE4SitWZz09)
(Password: 77777)
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
Thanks for suggesting these changes!  The linter got caught on 3 items, 2 items are from the suggestions. If you could accept the suggested commits and remove the extra line at 137 the linting error should clear.

**Comment by axelsimon** [comment]
> If you could accept the suggested commits and remove the extra line at 137 the linting error should clear.

Done! :)

**Review by ultrasaurus — APPROVED** [review]
LGTM -- will merge

**Comment by ultrasaurus** [comment]
oops -- forgot that requested changes needs approval by requestor!  So will leave merging to @TheFoxAtWork

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

**Pull request merged**